Customizing/global/lang/ilias_de.lang.local

```diff
@@ -1511,4 +1511,5 @@ gev#:#gev_training_admin_search_wip#:#in Bearbeitung
 gev#:#gev_training_admin_search_not_closed#:#noch nicht finalisiert
 gev#:#gev_filter_period_from#:#Von
 gev#:#gev_filter_period_to#:#Bis
-gev#:#gev_training_admin_search_period#:#Zeitraum
\ No newline at end of file
+gev#:#gev_training_admin_search_period#:#Zeitraum
+gev#:#gev_org_unit_no_superior#:#Sie sind in keiner Organisationseinheit Vorgesetzter.
```

Comment:
Bitte ändern zu: Dieser Bericht enthält keine Daten, da Sie in keiner Organisationseinheit Vorgesetzter sind.